### PR TITLE
Add new model for Merchant Center `CustomView` entity

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
This PR adds a new model for the `CustomView` new entity we're developing for the Merchant Center.

Also, I included another one (`LocalizedField`) in the commons package as I believe it can be used from other upcoming models.

### CODE REVIEW

**Review by tdeekens — APPROVED** [review]

**Comment by CarlosCortizasCT** [comment]
Building is currently failing but I don't understand how I can fix it:
```
$ pp build

> preconstruct build

🎁 error @commercetools-test-data/custom-view packages must have at least one entrypoint, this package has no entrypoints
🎁 info If you want to learn more about the above error, check https://preconstruct.tools/errors
🎁 info If the error is not there and you want to learn more about it, open an issue at https://github.com/preconstruct/preconstruct/issues/new
 ELIFECYCLE  Command failed with exit code 1.
```

File: models/custom-view/src/custom-view/generator.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const generator = Generator<CustomView>({
+  fields: {
+    createdAt: fake(getOlderDate),
+    defaultLabel: fake((f) => f.commerce.department()),
+    id: fake((f) => f.string.uuid()),
+    installedBy: null, // buildField
+    labelAllLocales: [
+      LocalizedField.random().build(),
+      LocalizedField.random().build(),
+    ],
```
Comment (CarlosCortizasCT) [inline models/custom-view/src/custom-view/generator.ts:20]: I'm not sure how should I implement these fields. I know we have the `buildFields` available in the transformers file, but I can't make it work.

Some help would be much appreciated.

File: models/custom-view/src/custom-view-permission/presets/manage-only-permissions.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import sampleSize from 'lodash/sampleSize';
+import upperFirst from 'lodash/upperFirst';
+import CustomApplicationPermission from '../builder';
+import { supportedManageOAuthScopes } from '../constants';
+
+const preset = (additionalPermission = '') => {
+  const resourceAccesses = entryPointUriPathToResourceAccesses(
+    CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH,
+    [additionalPermission]
+  );
```
Comment (emmenko) [inline models/custom-view/src/custom-view-permission/presets/manage-only-permissions.ts:13]: ℹ️ This requires the change in App Kit to be released: https://github.com/commercetools/merchant-center-application-kit/pull/3272

File: models/custom-view/src/custom-view/generator.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCustomView>({
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    ownerId: fake((f) => f.string.uuid()),
+    url: fake((f) => f.internet.url()),
+    defaultLabel: fake((f) => f.commerce.department()),
+    labelAllLocales: fake(() => [LocalizedField.random()]),
```
Comment (emmenko) [inline models/custom-view/src/custom-view/generator.ts:17]: The wrapping of `fake()` was missing, otherwise models cannot be built.

**Review by emmenko — APPROVED** [review]
🚀

**Pull request merged**